packages/core-base/src/context.ts

```diff
@@ -414,14 +414,11 @@ export function registerLocaleFallbacker(fallbacker: LocaleFallbacker): void {
 // Additional Meta for Intlify DevTools
 let _additionalMeta: MetaInfo | null = /* #__PURE__*/ null
 
-export const setAdditionalMeta = /* #__PURE__*/ (
-  meta: MetaInfo | null
-): void => {
+export const setAdditionalMeta = (meta: MetaInfo | null): void => {
```

Comment:
I think we swap to `/* #__NO_SIDE_EFFECTS__ */`, not remove `/* #__PURE__ */`
see the https://github.com/intlify/vue-i18n-next/issues/1599#issuecomment-1773776151
ref: https://github.com/javascript-compiler-hints/compiler-notations-spec/blob/main/no-side-effects-notation-spec.md